dsa-23au/java-dsa/heaps-trees/src/main/java/dev/codewithfriends/BSTMain.java

```diff
@@ -4,7 +4,7 @@
 
 public class BSTMain {
```

Comment:
Overall looking at everyone's pull requests regarding BSTs, I would definitely want to practice on implementing taraveral algorithms into my projects.